doc/api/process.markdown

```diff
@@ -137,13 +137,29 @@ Here is an example that logs every unhandled rejection to the console
         // application specific logging, throwing an error, or other logic here
     });
 
-For example, here is a rejection that will trigger the `'unhandledRejection'`
+For example, here is a rejection that will trigger the 'unhandledRejection'
```

Comment:
good point, I read this the wrong way, thinking he'd added it. It shouldn't be removed, this is a consistent style through the docs